images/virtualization-artifact/pkg/controller/cvi/internal/source/object_ref.go

```diff
@@ -264,6 +279,8 @@ func (ds ObjectRefDataSource) Validate(ctx context.Context, cvi *virtv2.ClusterV
 		return NewImageNotReadyError(cvi.Spec.DataSource.ObjectRef.Name)
 	case virtv2.ClusterVirtualImageObjectRefKindClusterVirtualImage:
 		return NewClusterImageNotReadyError(cvi.Spec.DataSource.ObjectRef.Name)
+	case virtv2.ClusterVirtualImageObjectRefKindVirtualDisk:
```

Comment:
NewDVCRDataSourcesForCVMI above should not be called for vd data source.

Reply:
Fixed.